src/options.json

```diff
@@ -0,0 +1,103 @@
+{
+  "additionalProperties": false,
+  "properties": {
+    "lessOptions": {
```

Comment:
sourceMap should be part of loader options, not Less options

No need describe all lessOptions, because in new release they will add more options

Reply:
Less too take this option i guess. Should I keep them both? 
Not all but I am thinking of covering max options. I actually wanted to keep the `additionalProperties` to `false`

Reply:
For lessOptions we should set `additionalProperties`, Less can change options in any time

Reply:
Ok then 👌